include/swift/AST/DeclAttr.def

```diff
@@ -712,7 +712,10 @@ DECL_ATTR(_expose, Expose,
   AllowMultipleAttributes | LongAttribute | UserInaccessible | ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove | ForbiddenInABIAttr,
   133)
 
-// Unused '134'
+SIMPLE_DECL_ATTR(_swiftPrivate, SwiftPrivate,
```

Comment:
We also have a `swift_private` attribute on the C++ side. Let me know if you think this would make this confusing (and potentially suggest an alternative name like `doNotExpose`, `swiftOnly`, `disableInterop`).

Reply:
I wonder if we could reuse the existing `@_expose` attribute here with a slightly extended syntax, e.g. `@_expose(!Cxx)`. Let's have a discussion about this.

Reply:
@DougGregor Was wondering if you have some opinions here.

Reply:
An alternative idea is to have "Nil" instead of the function name as a way to say not to expose it.

Reply:
Personally, I like `@_expose(!Cxx)` because it puts the "expose to another language" functionality under one attribute that we can document.